example/main.tf

```diff
@@ -0,0 +1,23 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+data "aws_availability_zones" "available" {}
```

Comment:
we don't need this since we have the var `var.availability_zones`

Reply:
Copy/pasta from my local where I didn't have the var.

I'll fix it up later.